src/ui/public/visualize/loader/visualize_data_loader.js

```diff
@@ -69,7 +69,7 @@ export class VisualizeDataLoader {
       return this._visData;
     }
     catch (e) {
-      this.props.searchSource.cancelQueued();
```

Comment:
`props` actually has a `searchSource` reference. Shouldn't we rather use that one, to not make more references to `Vis`?